mPowerSDK/Onboarding/APHOnboardingManager.h

```diff
@@ -38,4 +38,9 @@
 - (ORKTaskViewController *)instantiateOnboardingTaskViewController:(BOOL)signUp;
 - (ORKTaskViewController *)instantiateConsentViewController;
 
+// @protected
```

Comment:
d'oh, right. If this were in BridgeAppSDK I'd say put it in xxx_internal.h if we really want it to be protected, but in this context this is fine.